x-pack/plugins/observability_solution/inventory/public/hooks/use_kibana.tsx

```diff
@@ -5,19 +5,13 @@
  * 2.0.
  */
 
-import { useKibana } from '@kbn/kibana-react-plugin/public';
-import type { CoreStart } from '@kbn/core/public';
-import type { InventoryStartDependencies } from '../types';
-import type { InventoryServices } from '../services/types';
+import { CoreStart } from '@kbn/core/public';
+import { KibanaReactContextValue, useKibana } from '@kbn/kibana-react-plugin/public';
+import { InventoryStartDependencies } from '../types';
```

Comment:
Why are we switching from `import type` to just `import`?

Reply:
There is no real benefit of `import type` when the file being imported doesn't have any runtime code. But I'll add `import type` for consistency.